librz/bin/dwarf.c

```diff
@@ -2035,7 +2035,7 @@ static RzBinDwarfDebugAbbrev *parse_abbrev_raw(const ut8 *obuf, size_t len) {
 	return da;
 }
 
-RzBinSection *getsection(RzBinFile *binfile, const char *sn) {
+static inline RzBinSection *getsection(RzBinFile *binfile, const char *sn) {
```

Comment:
```suggestion
static RzBinSection *getsection(RzBinFile *binfile, const char *sn) {
```
no need for inline, compiler will automatically inline it if it can.